.cirrus.yml

```diff
@@ -1,12 +1,12 @@
 # Due to the dependency on encrypted variables, this will only work in a
 # specific repo
-only_if: $CIRRUS_REPO_OWNER == 'anarazel' && $CIRRUS_REPO_NAME == 'pg-vm-images'
+# only_if: $CIRRUS_REPO_OWNER == 'anarazel' && $CIRRUS_REPO_NAME == 'pg-vm-images'
 
 env:
   CIRRUS_LOG_TIMESTAMP: true
 
   # See gcp_project_setup.txt for setup instructions
-  GCP_PROJECT_MAIN: pg-ci-images
+  GCP_PROJECT_MAIN: pg-ci-images-dev
```

Comment:
To be able to merge these, this needs to be dropped

Reply:
reverted the change